### DISCUSSION TITLE
Readability issues arising from using `Long` for time representation

### DISCUSSION THREAD

**Discussion by raylras** [Strumenta/kolasu#334] [General]
When this code segment appears:

https://github.com/Strumenta/kolasu/blob/e06be725ec01ad44cbbfaf798ae013a4ce7ac31b/ast/src/commonMain/kotlin/com/strumenta/kolasu/parsing/Parsing.kt#L95

I need to guess the unit of `time`, whether it is in milliseconds, microseconds, or just seconds. This makes the code less readable.

Suggestion:
Replace `Long` with `kotlin.time.Duration`.

**Comment by ftomassetti** [comment]
Hello raylras, this is a good suggestion. We created a new issue to track this: https://github.com/Strumenta/kolasu/issues/335
We plan to tackle this in the Kolasu 1.6 branch which is under development.
In Kolasu 1.5 we will at least revise the documentation of the single methods (see https://github.com/Strumenta/kolasu/issues/336)
